src/core_landice/Registry.xml

```diff
@@ -289,6 +301,35 @@
 		            description="The fraction of the deepest grounding line depths over which the grounding line depth is averaged to obtain a reference grounding line depth for the 'temperature_profile' melt parameterization."
 		            possible_values="any positive real value"
 		/>
+                <!-- Options related to grounded glacier front submarine melt param. -->
+                <nml_option name="config_front_mass_bal_grounded" type="character" default_value="none" units="unitless"
+                            description="Selection of the method for computing the frontal mass balance of submarine grounded ice.  'none' sets the frontMassBal field to 0 everywhere.  'file' uses without modification whatever value was read in through an input or forcing file or the value set by an ESM coupler.  'constant' uses a prescribed constant melt rate. 'ismip6' uses the parameterization provided by ISMIP6 and requires an ocean thermal forcing field and subglacial hydrology."
```

Comment:
In the description, indicate what field is used for the 'file' option.

Reply:
I don't actually know the answer to that. I stole the description from config_basal_mass_bal_float before modifying, which doesn't include that information.

Reply:
It looks like 'none' and 'ismip6' are the only implemented option here.   Can you remove the other options from the possible_values and description?

Reply:
'file' has been removed. 'constant' has been replaced with 'uniform', which sets the face melt speed to the value given by config_uniform_face_melt_rate.